nemo_curator/scripts/semdedup/clustering.py

```diff
@@ -18,10 +18,10 @@
 
 import dask_cudf
 
+from nemo_curator import ClusteringModel
```

Comment:
We should keep this as semnatic dedup specific import (because clustering can be generic). So something like:

Reply:
It looks like `EmbeddingCreator`, `ClusteringModel`, and `SemanticClusterLevelDedup` are currently available as generic imports in NeMo Curator: https://github.com/NVIDIA/NeMo-Curator/blob/main/docs/user-guide/api/deduplication.rst

Should I change it to only be importable via `nemo_curator.semantic_dedup` instead?

Reply:
Hi @VibhuJawa for now I am going to merge as is (as I have several PRs related to dedupe that I would like to update with this PR), and I can create a separate PR if we decide to change the import logic. Thanks!